src/fable-elmish/README.md

```diff
@@ -9,12 +9,12 @@ Elmish abstractions have been carefully designed to resemble Elm's "look and fee
 
 ## Installation
 
+In a project directory with `dotnet-fable` CLI tools installed, type:
+
 ```shell
-npm install --save fable-core fable-powerpack fable-elmish
+dotnet fable add fable-powerpack fable-elmish
```

Comment:
Hehe, actually the `add` command is not working yet. It'll just invoke `npm install fable-powerpack`, then locate the package.json file (in the current or an upper directory) and use the naming convention to add a project reference using the new dotnet sdk command: `dotnet add reference ./node_modules/fable-powerpack/fable.powerpack.fsproj`.

About `dotnet fable` in general, if you have a nuget package named `dotnet-whatever`, you can add it to the project using a `DotNetCliToolReference` tag, the dotnet SDK lets you run the library using commands with the pattern `dotnet whatever`.